src/sidebar/components/MarkdownView.js

```diff
@@ -39,24 +35,17 @@ export default function MarkdownView({
     });
   }, [markdown]);
 
-  // Use a blank string to indicate that the content language is unknown and may be
-  // different than the client UI. The user agent may pick a default or analyze
-  // the content to guess.
-  const contentLanguage = '';
-
   return (
-    <div
-      className="w-full break-words cursor-text"
-      dir="auto"
-      lang={contentLanguage}
-    >
-      <div
-        className={classnames('styled-text', textClass)}
-        data-testid="styled-text"
-        ref={content}
-        dangerouslySetInnerHTML={{ __html: html }}
-        style={textStyle}
-      />
+    <div className="w-full break-words cursor-text">
+      <StyledText>
+        <div
+          className={classes}
+          data-testid="markdown-text"
+          ref={content}
+          dangerouslySetInnerHTML={{ __html: html }}
+          style={style}
+        />
+      </StyledText>
```

Comment:
Technically, we could do this as a single `StyledText` element without a child `div` (i.e. `StyledText` will take `dangerouslySetInnerHTML`) but I found that was messing with the overflow calculations on the `Excerpt` so left it as you see it.

Reply:
Might be worth a brief comment in the code about this.